cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest

```diff
@@ -9,7 +9,9 @@
 {% set cloud_config_volume = "" -%}
 
 {% if grains.cloud is defined -%}
-  {% set cloud_provider = "--cloud-provider=" + grains.cloud -%}
+  {% if grains.cloud != 'vagrant' -%}
```

Comment:
I am still getting out of my turkey coma, it's possible we can remove the vagrant cloud provider, just have not thought it through.